package.json

```diff
@@ -1,23 +1,46 @@
 {
   "name": "@mongodb-js/zstd",
-  "version": "1.2.2",
+  "version": "2.0.0-beta.0",
   "main": "index.js",
   "types": "index.d.ts",
   "repository": "https://github.com/mongodb-js/zstd",
   "files": [
     "index.d.ts",
     "index.js"
   ],
+  "dependencies": {
+    "bindings": "^1.5.0",
+    "node-addon-api": "^8.1.0"
+  },
   "license": "Apache-2.0",
   "devDependencies": {
+    "@mongodb-js/zstd": "^1.2.0",
+    "@typescript-eslint/eslint-plugin": "^8.11.0",
     "chai": "^4.5.0",
     "mocha": "^10.8.2",
     "prettier": "^3.3.3",
-    "standard-version": "^9.5.0"
+    "clang-format": "^1.8.0",
+    "sinon-chai": "^3.7.0",
+    "chai-subset": "^1.6.0",
```

Comment:
Do we expect to use these?

Let's filter some of the things that we can easily pull in if there is a need. (sinon-chai, but no sinon? 😅)

Reply:
Which shared tooling? 

we don't use subset in bson or kerberos (it is kind of a foot gun assertion)

Still, sinon-chai with no sinon doesn't make sense, I don't think we'll need either so let's just remove what is unused.

Reply:
chai-addons: https://github.com/mongodb-js/zstd/blob/NODE-6539/test/tools/chai-addons.js

I chose consistency with other repos over eliminating unnecessary tooling, so even though we don't currently use these packages, I included the config and the packages it depends on.  However, your comments on this PR indicate you'd prefer that I remove all unused tooling and dependencies, so I will.